dace/sdfg/utils.py

```diff
@@ -1295,6 +1295,10 @@ def inline_sdfgs(sdfg: SDFG, permissive: bool = False, progress: bool = None, mu
     from dace.transformation.interstate import InlineSDFG, InlineMultistateSDFG
 
     counter = 0
+
```

Comment:
This will be slow. Please do not use apply_transformations_repeated in this pass as it is designed to be scalable w.r.t. large SDFGs.